gradlew

```diff
@@ -86,7 +86,7 @@ APP_NAME="Gradle"
 APP_BASE_NAME=${0##*/}
 
 # Add default JVM options here. You can also use JAVA_OPTS and GRADLE_OPTS to pass JVM options to this script.
-DEFAULT_JVM_OPTS='"-Xmx64m" "-Xms64m"'
```

Comment:
this file is generated by gradle wrapper. It is only the memory used for the stub afaik.

For example I periodically will run something like
```
./gradlew wrapper --gradle-version=7.5.1
```
Did you see issues with the previous settings?

Reply:
RIght. This was an attempt to increase jvm heap size for gradle due to outOfMem error while generating dependency tree but later figured out that this is done via the gradle properties and not changing the defaults in the wrapper. Reverting this line. Thanks.